src/panels/config/dashboard/ha-config-navigation.ts

```diff
@@ -23,6 +23,7 @@ const PAGES: Array<{
   advanced?: boolean;
 }> = [
   { page: "core", core: true },
+  { page: "server_control", core: true, advanced: true },
```

Comment:
Restarting and shutting down Home Assistant should not be hidden by an advanced flag. It should be accessible to all.

Reply:
I have changed this. Good catch indeed. The config validation and partial reload is still hidden if advanced mode is off